client/src/utils/createApiRequest.js

```diff
@@ -0,0 +1,11 @@
+export default function createApiRequest(url, method, data) {
+  fetch(url, {
+    method,
+    body: data ? JSON.stringify(data) : null
```

Comment:
Would it work to short-circuit here, or no? I tend to prefer the `data && JSON.stringify(data)` format over the one-sided-ternary approach, when possible, but I've found it to be a little less predictable as to when it works.